pkg/querier/block_test.go

```diff
@@ -82,7 +82,7 @@ func TestBlockQuerierSeries(t *testing.T) {
 			sampleIx := 0
 
 			it := series.Iterator()
-			for it.Next() {
+			for it.Next() == chunkenc.ValFloat {
```

Comment:
I don't think we should stop iterating when we hit a non float value. To make this code future proof, I would iterate until `!= none` and then check the actual type if we only expect floats in this test.

_Same comment applies below too._

Reply:
replacing bool return value of advance function with chunkenc.ValueType and asserting float in the verify function since it only checks the float values.

Reply:
done